src/debugger.ts

```diff
@@ -146,6 +151,7 @@ export namespace Debugger {
     private _mode: IDebugger.Mode;
     private _modeChanged = new Signal<this, IDebugger.Mode>(this);
     private _currentLineChanged = new Signal<this, number>(this);
+    private _LinesCleared = new Signal<this, void>(this);
```

Comment:
The capital "L" looks like a type:

```suggestion
    private _linesCleared = new Signal<this, void>(this);
```